book/website/reproducible-research/ci/secrets.md

```diff
@@ -0,0 +1,7 @@
+## Pro Tips 
```

Comment:
I'm not sure what tips should we give here. So what do you think removing this file altogether, and rather open an issue that the current chapter could be extended with more info about secrets or using containers, or just as well writing about CD